.github/workflows/ci.yml

```
@@ -10,12 +10,15 @@ jobs:
       fail-fast: false
       matrix:
         java: [11, 8]
+        distribution: [adopt]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: olafurpg/setup-scala@v13
+      - uses: actions/setup-java@3
```

```suggestion
      - uses: actions/setup-java@2
```
I think 2 is the latest

---

https://github.com/actions/setup-java/releases/tag/v3.10.0 is the latest

---

`actions/setup-java@v3` it looked like it needs the `v`